financial-connections/src/main/java/com/stripe/android/financialconnections/ui/FinancialConnectionsSheetNativeActivity.kt

```diff
@@ -147,117 +130,43 @@ internal class FinancialConnectionsSheetNativeActivity : AppCompatActivity(), Ma
-                    BackHandler(navController, Pane.ACCOUNT_PICKER)
-                    AccountPickerScreen()
-                }
-                composable(NavigationDirections.success.destination) {
-                    LaunchedPane(Pane.SUCCESS)
-                    BackHandler(navController, Pane.SUCCESS)
-                    SuccessScreen()
-                }
-                composable(NavigationDirections.reset.destination) {
-                    LaunchedPane(Pane.RESET)
-                    BackHandler(navController, Pane.RESET)
-                    ResetScreen()
-                }
-                composable(NavigationDirections.attachLinkedPaymentAccount.destination) {
-                    LaunchedPane(Pane.ATTACH_LINKED_PAYMENT_ACCOUNT)
-                    BackHandler(navController, Pane.ATTACH_LINKED_PAYMENT_ACCOUNT)
-                    AttachPaymentScreen()
-                }
-                composable(NavigationDirections.networkingLinkSignup.destination) {
-                    LaunchedPane(Pane.NETWORKING_LINK_SIGNUP_PANE)
-                    BackHandler(navController, Pane.NETWORKING_LINK_SIGNUP_PANE)
-                    NetworkingLinkSignupScreen()
-                }
-                composable(NavigationDirections.networkingLinkLoginWarmup.destination) {
-                    LaunchedPane(Pane.NETWORKING_LINK_LOGIN_WARMUP)
-                    BackHandler(navController, Pane.NETWORKING_LINK_LOGIN_WARMUP)
-                    NetworkingLinkLoginWarmupScreen()
-                }
-                composable(NavigationDirections.networkingLinkVerification.destination) {
-                    LaunchedPane(Pane.NETWORKING_LINK_VERIFICATION)
-                    BackHandler(navController, Pane.NETWORKING_LINK_VERIFICATION)
-                    NetworkingLinkVerificationScreen()
-                }
-                composable(NavigationDirections.networkingSaveToLinkVerification.destination) {
-                    LaunchedPane(Pane.NETWORKING_SAVE_TO_LINK_VERIFICATION)
-                    BackHandler(navController, Pane.NETWORKING_SAVE_TO_LINK_VERIFICATION)
-                    NetworkingSaveToLinkVerificationScreen()
-                }
-                composable(NavigationDirections.linkAccountPicker.destination) {
-                    LaunchedPane(Pane.LINK_ACCOUNT_PICKER)
-                    BackHandler(navController, Pane.LINK_ACCOUNT_PICKER)
-                    LinkAccountPickerScreen()
-                }
-                composable(NavigationDirections.linkStepUpVerification.destination) {
-                    LaunchedPane(Pane.LINK_STEP_UP_VERIFICATION)
-                    BackHandler(navController, Pane.LINK_STEP_UP_VERIFICATION)
-                    LinkStepUpVerificationScreen()
-                }
+            BackHandler(true) {
+                viewModel.onBackClick(navController.currentDestination?.pane)
+                if (navController.popBackStack().not()) onBackPressedDispatcher.onBackPressed()
```

Comment:
is this overlapping with the feature provided by `NavigationManager.tryNavigateBack` ?

Reply:
Removed tryNavigateBack